data/brands/amenity/university.json

```diff
@@ -78,6 +78,18 @@
         "name": "University of Phoenix",
         "short_name": "UOPX"
       }
+    },
+    {
+      "displayName": "Stanford University",
+      "id": "stanforduniversity-d62529",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "university",
+        "brand": "Stanford University",
+        "brand:wikidata": "Q1889100",
```

Comment:
The universities tagged with `brand=*` are multi-campus for-profit university chains. Stanford has a few campuses besides the central campus, but these aren’t “brands”. Instead, the entry would need to go in the operator tree under data/operator/amenity/university.json.

NSI has [a different threshold of notability](https://github.com/osmlab/name-suggestion-index/wiki/Judge-Case#notability) than other projects like Wikipedia and Wikidata, because the purpose of NSI is to enforce consistent tagging across a large set of features in OpenStreetMap. Stanford and its satellite facilities are already mapped in OSM, and there aren’t very many of them. I suppose we could still add it to the operator tree, though the layout of that file won’t be able to handle an influx of other single-campus universities.

On the other hand, NSI is currently missing entries for Stanford Medicine, Stanford Health Care, Stanford Health Care Tri-Valley, and Stanford Medicine Partners, all of which are local healthcare brands that would have many more eligible features in OSM.

Reply:
Sure, I would be interested in contributing to the missing entries.

Reply:
can you guide me as to how exactly I can help